tests/test_suppression.py

```diff
@@ -1,11 +1,15 @@
 import numpy as np
 import pandas as pd
 import pytest
-import random
 
 import zentables as zen
```

Comment:
You can do `from .zentables import _do_suppression` here and avoid having `_do_suppression` as a top-level export, especially since `_do_suppression` begins with a `_` which indicates that it should never be exposed to users.